### PR TITLE
placesPhoto should throw error if neither maxwidth or maxheigh is declared

### PR DESCRIPTION
**Pull request by sverps**
placesPhoto should throw error if neither maxwidth or maxheight is declared

### CODE REVIEW

**Comment by sverps** [comment]
Thanks for the suggestions. I added the changes.

File: lib/internal/validate.js [hunk 1]
```diff
@@ -129,6 +129,23 @@ Validate.oneOf = function(names) {
   };
 };
 
+Validate.atLeastOneOfProperties = function(names) {
+  return function(value) {
+    if (!value) return value;
+
+    var quotedNames = [];
+    for (var name of names) {
```
Comment (stephenmcd) [inline lib/internal/validate.js]: Sorry I missed this line - does this style of for loop (`foo of bar`) change anything with regard to supported node versions? I don't think we've used that anywhere else throughout the library. Regardless we should probably change it just for the sake of consistency with the rest of the library.

Comment (sverps) [reply lib/internal/validate.js]: I verified it, and it is indeed an ES6 feature.
I swapped it out for a regular for loop.

**Comment by stephenmcd** [comment]
Thanks!

**Pull request merged**